app/components/payment-destination-display/payment-destination-display.tsx

```diff
@@ -40,14 +43,12 @@ export const PaymentDestinationDisplay = ({
 
   // we assume this is a bitcoin address or lightning invoice
   // not a username
-  const firstSix = destination.slice(0, 5)
-  const lastSix = destination.slice(-5)
-  const middle = destination.slice(5, -5)
+  const firstSix = destination.slice(0, -6)
```

Comment:
that doesn't sounds right. this select all the last 6 caracters. so the first 6 are no longer in bold.

Reply:
![image](https://github.com/GaloyMoney/galoy-mobile/assets/74586135/bc951106-5893-4309-ad39-0f1741a9afd7)